openstack_dashboard/dashboards/project/static/dashboard/project/containers/containers.controller.js

```diff
@@ -201,7 +203,7 @@
               {
                 key: 'name',
                 validationMessage: {
-                  exists: gettext('A container with that name exists.')
+                  exists: gettext('A container with that name exists. (Name must be unique within each region.)')
```

Comment:
I think @codyhammock also figured out that this is something that we can configure Ceph to allow.

Reply:
I'm not sure Ceph could solve this. And looks like Amazon s3 bucket name is globally unique, and the object stores of GCP and Azure require bucket name to be unique to the region. Please correct me if I'm wrong. @codyhammock